ODIN_II/SRC/string_cache.cpp

```diff
@@ -79,15 +79,15 @@ sc_lookup_string(STRING_CACHE * sc,
         return -1;
     }
     else {
-    hash = string_hash(sc, string) % sc->string_hash_size;
```

Comment:
the changes on this file seem only to be whitespace related. please remove this file from your commit